src/hip/hip_build_utils.cpp

```diff
@@ -45,6 +45,12 @@ bool IsHccCompiler()
     return isHcc;
 }
 
+bool IsClangXXCompiler()
```

Comment:
It is not used outside of this module. I recommend marking this static and removing declaration from the interface (.hpp).